helm/templates/frontend-env-config.yaml

```diff
@@ -5,6 +5,6 @@ metadata:
 data:
   env-config.js: |
     window.process_env = {
-      SUBSTRATE_TELEMETRY_URL: "ws://feed.telemetry.parity-stg.parity.io/feed",
+      SUBSTRATE_TELEMETRY_URL: "wss://feed.telemetry.parity-stg.parity.io/feed",
```

Comment:
Aah, did you find a way to allow the feed host to accept WSS instead of proxying through the frontend pod then?

Reply:
at the moment, I'm using the ingress-controller (just a reverse proxy like nginx) to expose the feed endpoint. I believe we wouldn't receive so much traffic from the clients so it wouldn't cause problems if we pass traffic though the ingress controller. Let's keep it this way for the stg environment to see how much traffic would come though the proxy. If so much traffic passes, we should scale the ingress controller which is quite easy.